main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -2395,14 +2395,20 @@ static void updateOfflineBox(final View view, final Geocache cache, final Resour
             final long diff = (System.currentTimeMillis() / (60 * 1000)) - (cache.getUpdated() / (60 * 1000)); // minutes
 
             final String ago;
-            if (diff < 15) {
+            if (cache.getUpdated() == 0L) {
+                ago = "";
+            } else if (diff < 15) {
                 ago = res.getString(R.string.cache_offline_time_mins_few);
-            } else if (diff < 50) {
+            } else if (diff < 60) {
                 ago = res.getQuantityString(R.plurals.cache_offline_about_time_mins, (int) diff, (int) diff);
             } else if (diff < (48 * 60)) {
                 ago = res.getQuantityString(R.plurals.cache_offline_about_time_hours, (int) (diff / 60), (int) (diff / 60));
-            } else {
+            } else if (diff < (365F / 12 * 24 * 60)) {
```

Comment:
...all the comparisons and quantity string arguments from here on would become much more readable by simply using the "days" variable and leaving away " x 24 x 60"